### PR TITLE
DJ Remove brackets from render functions to fix issue preventing the …

### PR DESCRIPTION
**Pull request by hdfeddanjackson**
…button component from rendering

### CODE REVIEW

File: components/atoms/button/Button.js [hunk 1]
```diff
@@ -21,13 +21,11 @@ const Button = props => {
 
     const isAnchor = url && (url.includes('http') || url.startsWith('#') || url.startsWith('mailto') || url.startsWith('/'))
 
-    const renderAnchor = () => {
+    const renderAnchor = () =>
         <a href={url} className={buttonClass}>{children}</a>
```
[the remark this answers is not in the document]
Comment (hdfedscottquested) [reply components/atoms/button/Button.js:25]: Much easier to read

[the remark this answers is not in the document]
Comment (hdfeddanjackson) [reply components/atoms/button/Button.js:25]: What if you want to include functionality in your render, the function is better than a variable.

[the remark this answers is not in the document]
Comment (hdfeddanjackson) [reply components/atoms/button/Button.js:25]: This also allows you to pass values using parameters into your render which you cannot do with variables.

[the remark this answers is not in the document]
Comment (hdfedscottquested) [reply components/atoms/button/Button.js:25]: Then perhaps add the brackets back in and remove them from the call...

```js
const renderAnchor = () => {
        <a href={url} className={buttonClass}>{children}</a>
}

return (
        isAnchor ? renderAnchor : renderButton
)
```

[the remark this answers is not in the document]
Comment (hdfedscottquested) [reply components/atoms/button/Button.js:25]: Stick with what is already there. My suggestion doesn't work.

**Review by hdfedscottquested — APPROVED** [review]
Approved

**Review by hd-darrenwhatford — APPROVED** [review]
Code makes sense to me.

Happy to approve.

**Pull request merged**